.github/workflows/prepare-release.yml

```
@@ -0,0 +1,86 @@
+name: Prepare Release
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        description: "Release version tag (e.g. v1.2.3)"
+        required: true
+      ref:
+        type: string
+        description: "Git ref from which to release"
+        required: true
+        default: "main"
+      do_publish_jars:
+        type: boolean
+        description: "Publish Java artifacts for staging"
+        required: true
+        default: "true"
+env:
+  NEXUS_SDK_REF: ${{ github.event.inputs.ref }}
+  NEXUS_SDK_TAG: ${{ github.event.inputs.tag }}
+
+jobs:
+  create_draft_release:
+    name: Create Github draft release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Audit gh version
+        run: gh --version
+
+      - name: Check for existing release
+        id: check_release
+        run: |
+          echo "::echo::on"
+          gh release view --repo "$GITHUB_REPOSITORY" "$NEXUS_SDK_TAG" \
+            && echo "::set-output name=already_exists::true" \
+            || echo "::set-output name=already_exists::false"
```

This is the new way to setting step output.

```suggestion
            && echo "already_exists=true" >> $GITHUB_OUTPUT \
            || echo "already_exists=false" >> $GITHUB_OUTPUT
```

---

This is taken from Java SDK repo, didn't want to deviate if I didn't have to